open_event/static/js/jquery/jquery.smartWizard.js

```diff
@@ -0,0 +1,489 @@
+/*
+ * SmartWizard 3.3.1 plugin
+ * jQuery Wizard control Plugin
+ * by Dipu
+ *
+ * Refactored and extended:
+ * https://github.com/mstratman/jQuery-Smart-Wizard
+ *
+ * Original URLs:
+ * http://www.techlaboratory.net
+ * http://tech-laboratory.blogspot.com
+ */
+
+function SmartWizard(target, options) {
+    this.target       = target;
+    this.options      = options;
+    this.curStepIdx   = options.selected;
+    this.steps        = $(target).children("ul").children("li").children("a"); // Get all anchors
+    this.contentWidth = 0;
+    this.msgBox = $('<div class="msgBox"><div class="content"></div><a href="#" class="close">X</a></div>');
+    this.elmStepContainer = $('<div></div>').addClass("stepContainer");
+    this.loader = $('<div>Loading</div>').addClass("loader");
+    this.buttons = {
+        next : $('<a>'+options.labelNext+'</a>').attr("href","#").addClass("btn btn-success"),
+        previous : $('<a>'+options.labelPrevious+'</a>').attr("href","#").addClass("btn btn-primary"),
+        finish  : $('<a>'+options.labelFinish+'</a>').attr("href","#").addClass("btn btn-default")
+    };
+
+    /*
+     * Private functions
+     */
+
+    var _init = function($this) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [All "var" declarations must be at the top of the function scope.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2663115097/issues/source?bid=3233864&fileBranchId=3324174#l33)